Example/ViewController.swift

```diff
@@ -35,7 +35,8 @@ class ViewController: UIViewController {
         let edgeInsets = UIEdgeInsets(top: 15, left: 15, bottom: 15, right: 15)
 
         let numberFormatter = FormNumberFormatter(
-            type: .custom(pattern: "***-***-****", replaceCharacter: "*")
+            pattern: "***-***-****",
```

Comment:
Could this become a problem for user's who'd want to use literal `*` in their number format?